lib/di/module/database_module.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:injectable/injectable.dart';
+import 'package:shared_preferences/shared_preferences.dart';
+
+@module
+abstract class DatabaseModule {
```

Comment:
[Optional] Perhaps we could call this `StorageModule` or `CacheModule`? I'm not sure if `SharedPreferences` counts as a database 😆